openapi.yaml

```
@@ -4,18 +4,33 @@ info:
+    ## Authentication
+    VolunChain uses wallet-based authentication with Stellar addresses.
+    - No passwords required
+    - JWT tokens for session management
+    - Support for user and organization profiles
 
-    All endpoints are namespaced under their respective version paths.
+    ## API Structure
+    This API uses a simple structure without versioning:
+    - `/api/` - Main API endpoints
+    - `/api/docs` - API documentation
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)